images/kube-api-rewriter/local/Dockerfile

```
@@ -1,5 +1,8 @@
+# Build kube-api-rewriter for local development purposes.
+# Note: it is not a part of the production build!
+
 # Go builder.
-FROM golang:1.21-alpine3.19 AS builder
+FROM golang:1.22-alpine3.19 AS builder
```

We need to use the version of the image containing a specific version of the go language to avoid downloading another version
```suggestion
FROM golang:1.22.7-alpine3.19 AS builder
```